focaccia.html

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Rachel J Show</title>
+<<<<<<< HEAD
```

Comment:
This probably got caught in a weird merge request, it was likely an accident, but it doesnt need to be here